scripts/exregional_run_analysis.sh

```diff
@@ -971,7 +971,11 @@ fi
 #
 # comment out for testing
 
-$APRUN ./gsi.x < gsiparm.anl > stdout 2>&1 || print_err_msg_exit "\
+$APRUN ./gsi.x < gsiparm.anl > stdout 2>&1 ;  errcode=$?
+echo "----------------------begin of stdout--------------"
+cat ./stdout  #log stdout whether gsi.x succeeds or not
+echo "----------------------end of stdout----------------"
```

Comment:
Do we need those two "echo" before and after cat?

Reply:
Those are not a must, but it helps to quickly locate stdout content in the log file (there are quite a lot of outputs after "end of stdout").  These two echoes should be harmless.